website/docs/Pipelines/snM3C/README.md

```diff
@@ -0,0 +1,92 @@
+---
+sidebar_position: 1
+slug: /Pipelines/snM3C/README
+---
+# snM3C Overview
+
+| Pipeline Version | Date Updated | Documentation Authors | Questions or Feedback |
+| :----: | :---: | :----: | :--------------: |
+| [snM3C_v1.0.0](https://github.com/yourusername/snM3C/releases) | August, 2023 | [Kaylee Mathews](mailto:warp-pipelines-help@broadinsitute.org) | Please file GitHub issues in the [WARP repository](https://github.com/broadinstitute/warp/issues) |
```

Comment:
this link https://github.com/yourusername/snM3C/releases doesnt work for me... is the yourusername supposed to auto fill in?

Reply:
I wish! No, that's just incorrect. Thank you for catching.